release-notes/opensearch.release-notes-2.1.0.md

```diff
@@ -0,0 +1,84 @@
+## 2022-07-05 Version 2.1.0 Release Notes
+
+
+### Breaking Changes in 2.1.0
+
+#### Remove Mapping types
+
+* [Type removal] Ignore _type field in bulk request ([#3505](https://github.com/opensearch-project/opensearch/pull/3505))
+* [Type removal] Remove type from BulkRequestParser (#3423) ([#3431](https://github.com/opensearch-project/opensearch/pull/3431))
+* [Type removal] _type removal from tests of yaml tests (#3406) ([#3414](https://github.com/opensearch-project/opensearch/pull/3414))
+
+#### Upgrades
+
+* [Upgrade] Lucene-9.3.0-snapshot-b7231bb (#3537) ([#3555](https://github.com/opensearch-project/opensearch/pull/3555))
```

Comment:
I don't think we need to mention these incremental snapshot commits (that have since been changed) in release notes. Instead we can mention the upgrade to 9.2.0 with https://github.com/opensearch-project/OpenSearch/commit/157e69330891b8133d56ab993b5022f2e8c6ee04.

Reply:
yes I think you are right, let me remove those other commits and just add upgrade to 9.2.0 with https://github.com/opensearch-project/OpenSearch/commit/157e69330891b8133d56ab993b5022f2e8c6ee04. Thanks @mch2